### PR TITLE
Remove Outtakes (the blooper song) from C&C.

### PR DESCRIPTION
**Pull request by GraionDilach**
I thought I'll get the hiding-music-from-UI done in time, but that didn't happened.

Outtakes shouldn't pop up in a playtest.

### CODE REVIEW

**Comment by dan9550** [comment]
This could be implemented as some kind of unlockable or since we have a campaign now, we could have a credits sequence at the end of that which it could play on.

File: mods/cnc/audio/music.yaml [hunk 1]
```diff
@@ -58,6 +58,5 @@ target: Target (Mechanical Man)
 j1: Untamed Land
 valkyrie: Ride of the Valkyries
 voic226m: Voice Rhythm
-outtakes: Outtakes
```
Comment (Mailaender) [inline mods/cnc/audio/music.yaml]: Maybe out-comment it instead.

**Comment by obrakmann** [comment]
It *would* be nice if this could play in the credits dialog, if we could ever make that work somehow. For the time being, :+1:

**Comment by pchote** [comment]
My scores.mix (from the original C&C95 disk) doesn't even have this track.  :+1: in any case.

**Pull request merged**